commands/build.go

```diff
@@ -629,6 +629,7 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions, debugConfig *debug.D
 	}
 
 	flags.StringVar(&options.printFunc, "call", "build", `Set method for evaluating build ("check", "outline", "targets")`)
+	cobrautil.BoolFunc(flags, "check", `Shorthand for "--call=check"`, callAlias(options, "check"))
```

Comment:
Can we do it so that the definition in here looks like:

```
flags.Var(&cobrautil.Smth{callAlias()}) , "check",  `Sho..`)
```
or
```
flags.Var(callAlias() , "check",  `Sho..`)
```

Reply:
Yea. The main purpose of this syntax was to mimic the upstream syntax.

https://pkg.go.dev/flag#FlagSet.BoolFunc

Reply:
All other flags use cobra-style definition, not stdlib, so we should stick to that.

Reply:
Done. I've changed it to directly expose `BoolFuncValue` instead so we can use `flags.Var` instead. `callAlias` then just returns `BoolFuncValue` to help it read a bit more easily.